rpc-server/src/utils.rs

```diff
@@ -171,10 +171,48 @@ where
     let config = Config::new(CompareMode::Strict).numeric_mode(NumericMode::AssumeFloat);
 
     // Sorts the values of the JSON objects before comparing them.
-    let read_rpc_json = json_sort_value(readrpc_response_json);
+    let read_rpc_json = json_sort_value(read_rpc_response_json);
     let near_rpc_json = json_sort_value(near_rpc_response_json);
 
     if let Err(err) = assert_json_matches_no_panic(&read_rpc_json, &near_rpc_json, config) {
+        // separate mismatching successful and failure responses into different targets
+        // to make it easier to find reasons of the mismatching
+
+        if read_rpc_response_success && near_rpc_response_success {
+            // Both services(read_rpc and near_rpc) have a successful response
+            // if data mismatch we are logging into `shadow_data_consistency_successful_responses`
+            // both response objects for future investigation.
+            tracing::warn!(target: "shadow_data_consistency_successful_responses",
```

Comment:
Introduction of couple of new targets really bothers me. Can you reuse the `shadow_data_consistency` we already have and instead define the cause of the warning in the log message itself, please?